llvm/test/CodeGen/AMDGPU/function-resource-usage.ll

```diff
@@ -0,0 +1,533 @@
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900 -enable-ipra=0 -verify-machineinstrs < %s | FileCheck -check-prefix=GCN %s
+
+; SGPR use may not seem equal to the sgpr use provided in comments as the latter includes extra sgprs (e.g., for vcc use).
```

Comment:
We should either fix the naming of these fields to say it's only the numbered SGPRs, or just always include the + vcc part?

Reply:
I applied the following renames:
the function level number of sgpr symbol went from `<function name>.num_sgpr` to `<function name>.numbered_sgpr`
remark went from `SGPRs` to `TotalSGPRs`
comment went from `NumSgprs` to `TotalNumSgprs`